test/case/infix_dhcp/dhcp_routes/test.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
-"""
-DHCP option 121 vs option 3
+"""DHCP option 121 vs option 3
 
-Verify DHCP option 121 (static routes) is used over option 3 and that the
-routes exist in the operational datastore.
+Verify that DHCP option 121 (classless static routes) is used over the
+older option 3 (default gateway) in the DHCP client, see RFC3442.  Use
+the routing RIB in the operational datastore to verify.
 
-Installing unrelated routes from a DHCP server should not affect already
-existing routes.  To verify this a canary route is set up in the client
-before initiating DHCP.  This canary route does not need to be reachable
-before a DHCP lease has been acquired.
+Installing routes from a DHCP server should not affect already existing
+(static) routes.  To verify this, a static canary route is set in the
+client before starting the DHCP client.  This canary route does not need
+to be reachable before a DHCP lease has been acquired.
```

Comment:
Perhaps change the last sentence to something like
_As a twist, this canary route has a next-hop (192.168.0.2) not reachable before a DHCP lease has been acquired._

And further down in the test steps, here are some thoughts.

"Verify 'client' has a route to 10.0.0.0/24 via 192.168.0.254"
=> "Verify 'client' has a route to 10.0.0.0/24 via 192.168.0.254 _(option 121)_"

"Verify 'client' has a default route via 192.168.0.254"
=> "Verify 'client' **does not have** a default route via 192.168.0.254 _(no option 3)_"
Or should it say _has a route_ as written before?

Reply:
Great stuff!  I had made similar changes, 

> - Verify 'client' has a route 10.0.0.0/24 via 192.168.0.254 (option 121)
> - Verify 'client' has default route via 192.168.0.254 (not use option 3)
> - Verify 'client' still has canary route to 20.0.0.0/24 via 192.168.0.2